ffi/CMakeLists.txt

```diff
@@ -49,9 +52,13 @@ add_library(llvmlite SHARED assembly.cpp bitcode.cpp core.cpp initfini.cpp
 # Bug reported upstream at: https://bugs.llvm.org/show_bug.cgi?id=47003
 # BROKEN: llvm_map_components_to_libnames(llvm_libs all)
 if ($ENV{LLVMLITE_SHARED})
-    set(llvm_libs LLVM)
+    message("ENV LLVMLITE_SHARED")
```

Comment:
I suspect you were using this for debugging CMake at some point - it should be removed.

Reply:
oh yeah my bad. I can remove any debug messages I used. I had completely forgot about that.